stdlib/marshal.gr

```diff
@@ -423,8 +423,11 @@ let marshal = (value, buf) => {
 @unsafe
 provide let marshal = value => {
   let valuePtr = fromGrain(value)
-  let buf = allocateBytes(size(valuePtr))
+  let size = size(valuePtr)
+  let buf = allocateBytes(size)
+  Memory.fill(buf + 8n, 0n, size)
```

Comment:
Remember when we introduced bytes/buffer and I asked "shouldn't we always zero it out"? 😜

Reply:
We do, actually! This is just the low-level case where that doesn't happen.